packages/lodestar-cli/src/commands/beacon.ts

```diff
@@ -33,6 +43,11 @@ export class BeaconNodeCommand implements ICliCommand {
       .command("beacon")
       .description("Start lodestar node")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-p, --preset [preset]", "Minimal/mainnet", "minimal")
+      .option("-s, --genesisState [params]", "Start chain from known state")
+      .option("--peer-id [peerId]", "peer id hex string or json file path")
+      .option("-e, --eth1", "interop for using fake eth1", "chain")
```

Comment:
This 'beacon' command is the main entry point for running a production beacon chain, we likely will want to keep the interop-related commands on the 'dev' command.